### ISSUE TITLE
FPS caps at 93 instead of 100 sometimes

### ISSUE THREAD

**Issue by shoober420** [ValveSoftware/halflife#1264]
I'm not sure what causes this. But way before the updates, I had this problem that my FPS would get capped at 93 instead of 100. I have to restart the game (sometimes more then once) for it to return to solid 100 FPS. If I don't my FPS will stay at 93. This is happening at the games menu, so there is no possibility that this is caused by the game stressing the system. Its at the start screen. It returns to normal after restarting the game. This issue has existed like 5 years back on my old system which was a Pentium 4 with a Geforce 3. Does anyone else experience this problem?

Windows 7 64-bit
Core2Duo E8400 @3.85GHz
Radeon 4890
4GBs DDR2
X-Fi Titanium

**Comment by shoober420** [comment]
This was indeed my AV when I used Windows. It was AVIRA. I honestly recommend not installing third party AV for a gaming machine, and just use Windows Defender and watch what you download.

**Comment by agrastiOs** [comment]
Indeed, Windows 10 has a great built-in antivirus too.

**Comment by shoober420** [comment]
@agrastiOs lolol, before I switched over too Linux when I used Windows 7, I wasn’t using any AV at all, just a firewall. Windows Defender does seem pretty light, but for more advanced users, I would tweak and disabled services, including Windows Defender if you watch what you download and don’t torrent. I remember using this site for service tweaks.

http://www.blackviper.com/

That and COMODO Killswitch to disable even more services. COMODO Firewall was bretty gud, but Windows Firewall is decent enough for most users. Another good site to check your security is Steve Gibson’s ShieldsUp.

https://www.grc.com/shieldsup
